src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -71,6 +71,7 @@ def load_controllers(mgrmodule):
 
 
 def _json_error_page(status, message, traceback, version):
```

Comment:
Maybe you should rename this function to ``json_error_page`` because it is not private anymore.